src/main/java/com/example/springmvcprodukty/ProductController.java

```diff
@@ -20,33 +20,26 @@ public ProductController(ProductRepository productRepository) {
     @GetMapping("/list")
     public String home(Model model, @RequestParam(required = false, name = "kategoria") ProductCategory category) {
```

Comment:
Jeżeli urle masz po angielsku (list, a nie lista), to parametry też raczej powinn być po angielsku